gputop-csv/gputop-csv.js

```diff
@@ -241,6 +296,7 @@ GputopCSV.prototype.update_features = function(features)
             var col_width = 0;
 
             if (this.pretty_print_csv_) {
+
```

Comment:
Please remove the newlines.

Reply:
Sorry for my mistake, I have removed this line.